### ISSUE TITLE
Request: Notification Sound

### ISSUE THREAD

**Issue by magnumchode** [Jerrkawz/google-voice-desktop-app#26]
Not sure how possible this is, but I really miss the old Google Voice extensions "Ding" sound when receiving a new text.  Would it be possible to implement this in the app?

I dug through the old extension's CSS to find the audio if it is possible and helps at all:
http://www.google.com/googlevoice/audio/bell.mp3

**Comment by brmonaghan** [comment]
Or maybe this.
https://drive.google.com/file/d/1_P4LdH_WVeEqP4-9TitfjL5vn5_q7DuD/view?usp=sharing
That's for hangouts, and I recorded this. @Jerrkawz will this help?

**Comment by AndersOhman** [comment]
I just switched over to your app from the Hangouts Chrome extension now that Google finally killed Voice SMS through there, and this would definitely be an appreciated and useful feature! Could this be easily implemented via the system that flags the icon with the unread message counter?

**Comment by brmonaghan** [comment]
Yes, the notifications feature is in the Voice program. All it does is when you receive a new message, it will show a toast notification and will NOT sound a message. I will fix that in my version (v1.1.9). Already shows the number of unread messages.
Thanks
BRYCE M

Sent from Mail<https://go.microsoft.com/fwlink/?LinkId=550986> for Windows 10

**Comment by watsonix** [comment]
Would be excited to change from the current sound which typically signifies "process complete" in MacOS to the one @brmonaghan suggested above which was in the last hangouts. Is that possible? +1 on badge count

**Comment by arooni** [comment]
seconded!  would love visual/sound notifications via this app

**Comment by ProphetPX** [comment]
please make it happen?  💯 👍
